modules/custom/express_content_edit_lock/express_content_edit_lock.module

```diff
@@ -262,12 +262,12 @@ function express_content_edit_lock_user_login(&$edit, $account) {
  * Prevents non-developers from logging in.
  */
 function express_content_edit_lock_form_user_login_alter($form, &$form_state) {
-  $dev_lock = variable_get('lock_user_dev');
-  if ($dev_lock === TRUE) {
+  $dev_lock = variable_get('lock_user_dev', FALSE);
+  if ($dev_lock == 1) {
     drupal_set_message(t('You will not be able to login to this site. It has been locked. Please contact the Web Support team. <a href="http://www.colorado.edu/webcentral/">@link_text</a>',
       array(
         '@link_text' => 'Visit Webcentral',
       )
-    ), 'warning');
+    ), 'error');
```

Comment:
I like this better as a warning since an error usually means the user did something wrong.